mypy.ini

```diff
@@ -21,16 +22,14 @@ exclude = (?x)(
 # *.extern modules that actually live in *._vendor will also cause attr-defined issues on import
 disable_error_code = attr-defined
 
-# - Avoid raising issues when importing from "extern" modules, as those are added to path dynamically.
-#   https://github.com/pypa/setuptools/pull/3979#discussion_r1367968993
+[mypy-pkg_resources._vendor.*,setuptools._vendor.*,setuptools.config._validate_pyproject.*,setuptools.tests.*]
+# Even when excluded there might be problems: python/mypy/issues/11936#issuecomment-1466764006
```

Comment:
Can this be a clickable link just for convenience? (even Refined GitHub doesn't understand this one as a GitHub location, and I doubt IDEs would either).
```suggestion
# Even when excluded there might be problems: https://github.com/python/mypy/issues/11936#issuecomment-1466764006
```